### PR TITLE
Add doc changes for PipelinesAsCode

### PR DESCRIPTION
**Pull request by savitaashture**
# Changes

Add doc changes for Pipelines As Code

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Run `make test lint` before submitting a PR
- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

### CODE REVIEW

File: docs/OpenShiftPipelinesAsCode.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        error-log-snippet: "true"
+        hub-catalog-name: tekton
+        hub-url: https://api.hub.tekton.dev/v1
+        remote-tasks: "true"
+        secret-auto-create: "true"
+        secret-github-app-token-scoped: "true"
+      targetNamespace: openshift-pipelines
+    ```
+
+- Check the status of installation using following command:
```
Comment (mramendi) [inline docs/OpenShiftPipelinesAsCode.md:44]: "of the installation" - grammar

File: docs/OpenShiftPipelinesAsCode.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    ```sh
+    kubectl get openshiftpipelinesascodes.operator.tekton.dev
+    ```
+
+## PipelinesAsCode Config 
+
+There are some fields which you can define on OpenShiftPipelinesAsCode CR to configure the behaviour of the PipelinesAsCode
```
Comment (mramendi) [inline docs/OpenShiftPipelinesAsCode.md:52]: Probably this is not the expected way and one should use TektonConfig instead?

File: docs/OpenShiftPipelinesAsCode.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+### Properties (Mandatory)
+
+ - `targetNamespace`
+
+    Setting this field to provide the namespace in which you want to install the PipelinesAsCode component.
```
Comment (mramendi) [inline docs/OpenShiftPipelinesAsCode.md:58]: "Set this field to provide..."

File: docs/TektonConfig.md [hunk 4]
```diff
@@ -278,6 +301,40 @@ spec:
       default-tekton-hub-catalog: tekton
 ```
 
+### OpenShiftPipelinesAsCode
+PipelinesAsCode section allows user to customize the Pipelines as Code features. This allow user to customize the values in configmaps.
```
Comment (mramendi) [inline docs/TektonConfig.md:305]: "The" at the start - grammar, also "allows the user" or "allows you" (both times). Also "to customize values in configmaps" does not sound clear. I suggest "When you change the TektonConfig CR, he Operator automatically applies the settings to custom resources and configmaps in your installation"

File: docs/TektonConfig.md [hunk 5]
```diff
@@ -278,6 +301,40 @@ spec:
...
+
+Some of the fields have default values, so operator will add them if user haven't passed in CR. Other fields which
```
Comment (mramendi) [inline docs/TektonConfig.md:307]: "if the user hasn't" - grammar

File: docs/TektonConfig.md [hunk 6]
```diff
@@ -278,6 +301,40 @@ spec:
...
+doesn't have default values won't be passed unless user specify them in CR. User can find those [here](https://pipelinesascode.com/docs/install/settings/#pipelines-as-code-configuration-settings).
```
Comment (mramendi) [inline docs/TektonConfig.md:308]: don't have default values - grammar
unless the user specifies them - grammar

File: docs/TektonConfig.md [hunk 7]
```diff
@@ -278,6 +301,40 @@ spec:
...
+          ]*)?(?P<error>.*)
+        error-log-snippet: "true"
+        hub-catalog-name: tekton
+        hub-url: https://api.hub.tekton.dev/v1
+        remote-tasks: "true"
+        secret-auto-create: "true"
+        secret-github-app-token-scoped: "true"
+```
+
+**NOTE**: OpenShiftPipelinesAsCode is currently available for OpenShift Platform only.
```
Comment (mramendi) [inline docs/TektonConfig.md:337]: for the OpenShift Platform only

File: docs/OpenShiftPipelinesAsCode.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+### Properties
+
+The fields have default values so even if user have not passed them in CR, operator will add them. User can later change
```
Comment (mramendi) [inline docs/OpenShiftPipelinesAsCode.md:62]: "The user" in both cases, also "has not" - grammar

File: docs/OpenShiftPipelinesAsCode.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+---
+linkTitle: "OpenShiftPipelinesAsCode"
+weight: 9
+---
+-->
+# OpenShiftPipelinesAsCode
+
+OpenShiftPipelinesAsCode custom resource allows user to install and manage [OpenShift PipelinesAsCode](PipelinesAsCode).
+
+It is recommended to install the component through [TektonConfig](./TektonConfig.md).
```
[the remark this answers is not in the document]
Comment (savitaashture) [reply docs/OpenShiftPipelinesAsCode.md]: Updated :+1: 
Thank you

File: docs/OpenShiftPipelinesAsCode.md (hunk 2 above)
Comment (savitaashture) [reply docs/OpenShiftPipelinesAsCode.md]: Yes you are right modified sentence PTAL
Thank you

**Pull request merged**